src/styles/main.scss

```diff
@@ -1,3 +1,24 @@
-body {
-  margin: 0;
+@import "./utils/variables";
+
+@import "./blocks/body.scss";
+@import "./blocks/header.scss";
+@import "./blocks/card.scss";
+@import "./blocks/stars.scss";
+
+a {
+  text-decoration: none;
+}
+
+.main {
+  display: flex;
+}
+
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  padding-top: 50px;
+  padding-bottom: 71px;
+  min-width: 950px;
+  max-width: 951px;
```

Comment:
Maybe we can improve it somehow? Looks confusing

Reply:
Not fixed. All values must be rounded and divisible by 8 or 4 without a remainder. If you use such strange styles somewhere,  there's a centering or indentation bug somewhere